Server_Side_TLS.mediawiki

```diff
@@ -202,6 +206,14 @@ This configuration is compatible with a number of very old clients, and should b
 ! Editor
 ! Changes
 |-
+| style="text-align: center;" | 5.7
```

Comment:
Up on line 198 and 200, you need to update the JSON file link from 5.6 to 5.7.

Reply:
I fixed this but forgot to commit it so it's in #293